src/public/services/AdminSubmissionsService.ts

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios'
+
+import { EncryptedSubmissionDto, SubmissionMetadataList } from 'src/types'
+import {
+  SubmissionCountQueryDto,
+  SubmissionMetadataQueryDto,
+  SubmissionResponseQueryDto,
+} from 'src/types/api'
+
+import { ADMIN_FORM_ENDPOINT } from './AdminFormService'
+
+/**
+ * Counts the number of submissions for a given form
+ * @param urlParameters Mapping of the url parameters to values
+ * @returns The number of form submissions
+ */
+export const countFormSubmissions = async ({
+  formId,
+  startDate,
+  endDate,
+}: SubmissionCountQueryDto): Promise<number> => {
+  const queryUrl = `${ADMIN_FORM_ENDPOINT}/${formId}/submissions/count`
+  if (startDate && endDate) {
+    return axios
+      .get(queryUrl, {
+        params: { startDate, endDate },
+      })
+      .then(({ data }) => data)
+  }
+  return axios.get(queryUrl).then(({ data }) => data)
+}
+
+/**
+ * Retrieves the metadata for either a page of submission or a single submissionId if submissionId is specified
+ * @param formId The id of the form to retrieve submission for
+ * @param submissionId The id of the specified submission to retrieve
+ * @param pageNum The page number of the responses
+ * @returns The metadata of the form
+ */
+export const getFormsMetadata = async ({
+  formId,
+  submissionId,
+  pageNum,
+}: SubmissionMetadataQueryDto): Promise<SubmissionMetadataList> => {
+  const params = submissionId ? { submissionId } : { page: pageNum }
```

Comment:
add comment about how this is mutually exclusive, or split this into two functions so there is no ambiguity?

Reply:
good point - will do!